core/appeal/service.go

```diff
@@ -314,6 +314,31 @@ func (s *Service) Create(ctx context.Context, appeals []*domain.Appeal, opts ...
 	}
 
 	for _, a := range appeals {
+		if a.Status == domain.AppealStatusRejected {
```

Comment:
have we done the same for `a.Status == domain.AppealStatusApproved`?

Reply:
Yes. that is already in place [here](https://github.com/goto/guardian/blob/main/core/appeal/service.go#L292).